src/screens/InitWallet.js

```diff
@@ -316,7 +316,7 @@ class LoadWordsScreen extends React.Component {
                 />
                 <Text style={this.style.label}>
                   {this.state.words.length}
-/
+                  /
```

Comment:
Blank space does not get rendered, right?

Reply:
I'm sorry, this change was added when I was thinking of changing the eslint configuration to the react-native-community (we use airbnb with some customs), but reversed it because we wouldn't gain much other than many changes on the src folder which should not be the focus of this PR.

I'll remove it.